app/views/users/_roles.html.slim

```diff
@@ -7,5 +7,9 @@
       label.radio
         = radio_button_tag :role, role == 'all' ? '' : role, params[:role] == role
         = role.capitalize
-
+    h4 Coach availability
+    label.radio
+      = radio_button_tag :availability, true
+      | Availability in the upcoming season
```

Comment:
As for the term "upcoming": shall we replace it with the actual year? (`Season#name`)